monitor/census.go

```diff
@@ -453,6 +464,12 @@ func InitCensus(nodeType, nodeID, version string) {
 			TagKeys:     baseTags,
 			Aggregation: view.Distribution(0, .10, .20, .50, .100, .150, .200, .500, .1000, .5000, 10.000),
 		},
+		{
+			Name:        "download_time_seconds",
+			Measure:     census.mDownloadTime,
```

Comment:
```suggestion
			Measure:     census.mDownloadTime,
			Description: "Download (from orchestrator) time",
```
I think this description is used in /metrics output

Reply:
fixed in 604fab7